### PR TITLE
SP2-91 create csv controller class

### PR DESCRIPTION
**Pull request by 0rangeBlaze**
I have added the documentation for the CsvSerializer. Please check if it is understandable and there are no typos or any other issue.

### CODE REVIEW

File: Documentation/Heatington/Controllers/Serializers/CsvController.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Constructor
+
+`public CsvData(List<string[]> data, string[]? header = null)`
+
+data: a table of deserialized data as plain text<br>
+header: holds entries of the CSV header
+
+Methods:
+### Create\<T\>
```
Comment (pkyeager) [inline Documentation/Heatington/Controllers/Serializers/CsvController.md:53]: I may be wrong, but I do believe this format is better for readabilty. 
```suggestion
### Create\<T\>

`public static CsvData Create<T>(List<T> data, string[]? header = null)`

Parameters:

- `data` (List<T>): this is a list of objects that you want to represent as CSV data. 
- `header` (string[]?, optional): a default header to be used. Default is null.

Returns:

- CsvData: Returns the `CsvData` created from `data` and `header`.
```

File: Documentation/Heatington/Controllers/Serializers/CsvController.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+`public static CsvData Create<T>(List<T> data, string[]? header = null)`
+
+data: a list of objects to be represented as CSV data<br>
+header: a default header to be used<br>
+Returns the CsvData created from data and header
+
+If header is null a header is created from the list of public properties of the T type. The CSV data will contain the values of the public properties of the T type turned to string via .ToString(), in case a property is null an empty string will be used as an entry.
```
Comment (pkyeager) [inline Documentation/Heatington/Controllers/Serializers/CsvController.md:61]: I may be wrong, but I do believe this format is better for readabilty. The same is for another methods in this documentation.
```suggestion
### Create\<T\>

`public static CsvData Create<T>(List<T> data, string[]? header = null)`

Parameters:

- `data` (List<T>): this is a list of objects that you want to represent as CSV data. 
- `header` (string[]?, optional): a default header to be used. Default is null.

Returns:

- CsvData: Returns the `CsvData` created from `data` and `header`.
```

```

**Comment by magwrap** [comment]
rebased new changes

**Review by frske — APPROVED** [review]
Nothing more to review here. This PR can be closed imo

**Review by magwrap — CHANGES_REQUESTED** [review]
Emm now all the documentation is held in docs/ directory. 
- Move the CsvController.md from Documentation/Heatington/Controllers/ to docs/Heatingtion/Services/CsvSerializer.md
The document is only for serializer not csv controller right? So the name should match.

Sorry Dani :(

**Review by magwrap — APPROVED** [review]
All good

**Pull request merged**